umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -465,6 +465,48 @@
       (update :Type #(update-url-with-default-values :Type %))
       (update :URLValue #(update-url-with-default-values :URLValue %))))
 
+(defn update-related-url-to-online-resource
+  "Takes a related-url and returns an Online Resource map with a couple of fields copied over."
+  [related-url]
+  {:Description (:Description related-url)
+   :Name (:Type related-url)
+   :Linkage (:URL related-url)})
+
+(defn update-contact-info-1_3_1->1_3_2
+  "For each contact info change the RelatedUrl map to an OnlineResource map."
+  [contact-info]
+  (-> contact-info
+      (assoc :OnlineResources (seq (map #(update-related-url-to-online-resource %)
```

Comment:
Nitpick: it might be more readable to put the `(seq ...)` into a `let` block

Reply:
updated